README.md

```diff
@@ -29,6 +29,12 @@ After cloning the repo run
 cargo install --path .
 ```
 
+or using alias from `.cargo/config.toml`
```

Comment:
Would that be for testing? If we do that then it will generate the files in the current folder which I'm thinking wouldn't be desirable unless your testing (then you have to delete it). Maybe this is something we should incorporate into the init command instead to add it to the "solutions" repo.